### ISSUE TITLE
ElectronJS

### ISSUE THREAD

**Issue by louanfontenele** [Abu3safeer/crunchyroll-dl-gui#2]
There is a anyway to create it on electronjs to build in all plataforms? i already have the frontend.

**Comment by Abu3safeer** [comment]
I never used electronjs, I wanted to build this on electronjs, but as I know electronjs  will be very big file, even if it very simple application. 

do you have any experience in electronjs ?

**Comment by louanfontenele** [comment]
I don't have yet, i try to study it to make it.

**Comment by Abu3safeer** [comment]
What I need to know is (how much approximately will be the file size?).
Because windows version is only few kilobytes.

**Comment by louanfontenele** [comment]
depends of the app and the code, i already created the UI, we need just to implement it for MAC OS, Windows and Linux using ElectronJS, if we put on Web will be not good.
https://louanbastos.github.io/crunchyroll-dl-en/ <-- the UI

**Comment by Abu3safeer** [comment]
Web means "server-side" that means the server will process all tasks, which is not good idea.
Electronjs is good idea, because it runs on client device, that means no server needed.

but I still have the question, the size, how much? 
![image](https://user-images.githubusercontent.com/12091003/57941026-743d0000-78d6-11e9-9394-4d1ca4bfe865.png)

Can it beat my application size?

**Comment by louanfontenele** [comment]
I think the file will be more bigger than 26 kb because of the code and the design :/ but i think it'll not be more than 100MB

**Comment by louanfontenele** [comment]
I prefer ElectronJS because of the ease of running on multiple platforms. In the case on MAC, Windows and Linux

**Comment by Abu3safeer** [comment]
@louanbastos That is correct, Cross-platform is the best feature that would make any developer select it over any other app.

**Comment by louanfontenele** [comment]
Yes, so we just need to try to create for ElectronJS

**Comment by Abu3safeer** [comment]
I don't program in Electron js so I will pass.
